src/gerbil/core/sugar.ss

```diff
@@ -146,7 +146,10 @@ package: gerbil/core
       ((_ message detail ...)
        (stx-string? #'message)
        (apply raise-syntax-error #f (stx-e #'message) stx
-              (syntax->list #'(detail ...)))))))
+              (syntax->list #'(detail ...))))))
+
+  (defrules compilation-target? ()
+    ((_ sym) (eq? (gx#current-compilation-target) 'sym))))
```

Comment:
this is a little dangerous, if you use it without the expander loaded it can crash.

We need to add current-compilation-target to the expander-runtime exports and you need to put this macro in a module that is imported at the prelude +1 phi.

Feel free to create a new inner module around here.

Reply:
Where are those expander-runtime exports? 

What's wrong with this module, is it not +1? Does it just not belong there?

Reply:
It is not that it is not +1, it is also 0 which allows it to leak yo phi=0 where the expander is not necessary available.

Reply:
Also please dont use the fully qualified name to subvert the module system.

Put it in the right place so that it is visible, this is an ugly and dangerous hack.